### ISSUE TITLE
Poor contrast within inactive buttons

### ISSUE THREAD

**Issue by rachels01** [carbon-design-system/carbon#8273]
Contrast of inactive button text and background does not past a contrast check. It is likely difficult for many users to read. 

![image](https://user-images.githubusercontent.com/81767728/113323780-60d98900-92e4-11eb-8a2a-052265612f83.png)

![image](https://user-images.githubusercontent.com/81767728/113323811-67680080-92e4-11eb-979f-8ea5c771ec67.png)

-->

## Detailed description

> What version of the Carbon Design System are you using?
v10

> What did you expect to happen?
Better contrast between button background and text

> What happened instead?
Button contrast does not meet accessibility standards

> What WCAG 2.1 checkpoint does the issue violate?
Contrast

## Steps to reproduce the issue

1. Visit a form page
2. Navigate to an inactive button 

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/main/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/main/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
- Notes

**Comment by joshblack** [comment]
Hi there @rachels01! 👋 

Per WCAG AA 2.1 guidelines, disabled states do not have to meet a minimum contrast ratio. Per 1.4.3: https://www.w3.org/TR/WCAG21/#contrast-minimum

> Text or images of text that **are part of an inactive user interface component**, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

Definitely understand that the contrast could be increased, but this is currently not a contrast violation for WCAG AA 2.1 requirements 👍

**Comment by rachels01** [comment]
@joshblack thanks for the quick reply! When I check the contrast ratio, I find that it is 2.04:1, which seems to be less than the 3:1 WCAG AA 2.1 standards. Is that not correct?

**Comment by joshblack** [comment]
@rachels01 basically the 1.4.3 rule states that inactive user interface components do not need to hit contrast standards. Hope this helps!
